model/model.py

```diff
@@ -184,45 +189,46 @@ def validate_batch(self):
 
     # TODO funktionen gör alldeles för mycket,
     # dela upp utskrift, beräkning och träning
-    def train(self):
+    def train(self, use_pretrained_net=False):
         """ Trains the model on the dataset """
         print("Starting training...")
 
-        if self.use_pretrained:
+        if self.use_pretrained and \
```

Comment:
Also, what does this actually mean? This pull request doesn't change behavior of the embedding matrix right? So why is it required that  self.use_pretrained_net == use_pretrained_net ? And why are there two separate variables for use_pretrained_net?